src/index.html

```diff
@@ -1,12 +1,34 @@
+<!-- #region head -->
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>Landing Page - B&O</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="./images/favicon/webicon.png"
```

Comment:
In your deployment, the favicon is missing:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/49671bb2-76ce-4bd3-90df-86be0497e403)
This is probably the same issue we talked about with the leading slash in the src

Reply:
Fixed!